### ISSUE TITLE
Classification examples

### ISSUE THREAD

**Issue by AAYamoldin** [OML-Team/open-metric-learning#225]
Hello. Could you add examples of Metric Learning Classification?
For low experience users in Data Science

**Comment by AlekseySh** [comment]
@AAYamoldin Hi! Do you mean using classification losses (like ArcFace) during the training?

**Comment by AAYamoldin** [comment]
As I understand, classical classification and Metric Learning are the same approaches. 
Only different is in loss function: classification - SoftMax, ArcFace and so on. For the Metric Learning - Triplet loss and so on.
So If I want to use ArcFace or SoftMax it will no "Metric Learning" and for this reason there is no examples for classification in OML?

**Comment by AlekseySh** [comment]
You can use ArcFace in ML pipeline. Here is an example: https://open-metric-learning.readthedocs.io/en/latest/examples/pml.html# (Training with loss from PML)

Is this answer to your question?

**Comment by AAYamoldin** [comment]
Yes, thank you
